metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -667,16 +674,26 @@ def _convert_value(param):
         "(run-id *{run_id}*).".format(name=obj.workflow_name, run_id=run_id),
         bold=True,
     )
+    try:
 
-    run_url = (
-        "%s/%s/%s" % (UI_URL.rstrip("/"), obj.flow.name, run_id) if UI_URL else None
-    )
-
-    if run_url:
```

Comment:
we do not want to remove the url to Metaflow UI in favour of Argo UI though? preferably we keep both, with tidy messaging as introduced in https://github.com/Netflix/metaflow/pull/1934/files#diff-27da6198697d250e8ed1cea087795ecbbf2a0e8e8e0861dd51c1f0062c2f0c41R653
which is a somewhat overlapping PR to this one.